app/src/main/java/com/keylesspalace/tusky/components/timeline/viewmodel/CachedTimelineViewModel.kt

```diff
@@ -289,6 +289,14 @@ class CachedTimelineViewModel @Inject constructor(
         }
     }
 
+    override fun saveReadingPosition(statusId: String) {
+        accountManager.activeAccount?.let { account ->
+            Log.d(TAG, "Saving position at: $statusId")
+            account.lastVisibleHomeTimelineStatusId = statusId
```

Comment:
TimelineFragment is used for more timelines than the home timeline (?)

Reply:
It is.

Which kind of timelines is CachedTimelineViewModel used for though...?

Reply:
Hint: https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt#L102-L108